### ISSUE TITLE
Curious on progress

### ISSUE THREAD

**Issue by kegiskeg** [plunkettscott/interact-sound#25]
Has the playOnEntity been updated so it will fade out from how far away you are from the sound? Been looking for this feature with interact-sound.

**Comment by plunkettscott** [comment]
No, it has not. Spatial audio is very much a thing I'm looking to implement, but it won't be in `interact-sound`, it'll be in the replacement package for `interact-sound` which has not been released yet.

Anything else you're missing in interact-sound?

**Comment by kegiskeg** [comment]
No your resource is perfect right now! Excited for that replacement package.
